code/modules/clothing/head/helmet.dm

```diff
@@ -359,6 +359,16 @@
 	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH | PEPPERPROOF
 	strip_delay = 80
 
+/obj/item/clothing/head/helmet/elder_atmosian
+	name = "Elder Atmosian Helmet"
+	desc = "A superb helmet made with the thoughest and rarest materials avaiable to man."
+	icon_state = "knight_greyscale"
+	inhand_icon_state = "knight_greyscale"
+	armor = list("melee" = 15, "bullet" = 10, "laser" = 20, "energy" = 20, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 40, "acid" = 40)
```

Comment:
doesnt use the armor defines, also could you please keep the helmet's and chestplate's armor values the same

Reply:
ah i missed this thanks. i'll change the numbers too